### DISCUSSION TITLE
Tuner(s) not found by scheduler

### DISCUSSION THREAD

**Discussion by jchiso** [OpenSageTV/Community#119] [OpenDCT]
I'm not really sure if this is an OpenDCT issue, but a couple on months ago my main server started having issues capturing from network encoders hosted on an OpenDCT instance, running on Ubuntu 21.04 server.  Curiously, if I schedule a capture I get failures and error messages.  If I tune to an program on live TV (and set to record) the tuning and capture work as expected.
There just seems to be an issue with Scheduled Recordings.
Any clues as to what is introducing the discrepancy?  This system worked flawlessly for over two years before these issues began ...

**Comment by trk2** [comment]
Do you have multiple encoders?  Is it possible that live tuning happens to select a working encoder where as the scheduler selects a non-working encoder per whatever merit you have set?

**Reply by jchiso** [reply]
I do have multiples encoders, but (according to the error messages) the failure happens on the same encoder that is selected by live tuning.  Also, if I do a tuning via Channel Setup, each of the encoders produces output.
I will try setting Encoder Merit to see if there is an issue there ...

**Reply by jchiso** [reply]
I tried changing the Encoder Merit settings for the two allocated capture devices.  Each experienced the same error when attempting to execute a scheduled capture.  Each also works when executing a live viewing.  I was even able to do a live tuning on one capture device while the other was engaged in the failed recording mode.
